packages/ui/src/components/mermaid.tsx

```diff
@@ -23,6 +25,7 @@ export const Mermaid: React.FC<{
 
   useEffect(() => {
     mermaid.initialize({
+      maxTextSize: 500000,
```

Comment:
```suggestion
      maxTextSize: 5000000,
```

Should we do something larger? AFAIK this is to prevent slow rendered, but I guess ignition users, being devs, would have ok-ish computers.